frontend/graph/impl/src/main/java/com/virtuslab/gitmachete/frontend/graph/impl/repository/RepositoryGraphBuilder.java

```diff
@@ -188,8 +188,8 @@ private BranchItem createBranchItemFor(
       GraphItemColor graphItemColor,
       @NonNegative int indentLevel) {
     RelationToRemote relationToRemote = branch.getRelationToRemote();
-    Option<IManagedBranchSnapshot> currentBranch = repositorySnapshot.getCurrentBranchIfManaged();
-    boolean isCurrentBranch = currentBranch.isDefined() && currentBranch.get().equals(branch);
+    @Nullable IManagedBranchSnapshot currentBranch = repositorySnapshot.getCurrentBranchIfManaged();
```

Comment:
Annotations are not needed for local variables. Additionally consider using `val` as it's a log type name and can be easily read by looking at the right hand side of the initialization..